.travis.yml

```diff
@@ -28,12 +28,12 @@ before_install:
 - cabal --version
 
 install:
-- cabal new-update -v hackage.haskell.org
-- cabal new-configure --enable-tests --write-ghc-environment-files=always
-- cabal new-build --only-dependencies -j
+- cabal new-update -v
+- cabal new-configure --enable-tests --disable-optimization --write-ghc-environment-files=always --jobs=2
```

Comment:
Why `--jobs=2` instead of `-j`?

Reply:
`-j` means `--jobs=NCPUS`. I was trying to see if 2 made things go any faster, since Travis images have one core visible.

Reply:
👍 thanks!